### DISCUSSION TITLE
Walkthrough for how to update the running miner

### DISCUSSION THREAD

**Discussion by MIKEBURNA** [swar/Swar-Chia-Plot-Manager#349] [Q&A]
Can some please be so kind. Please describe it as if you were explaining it to your little brother / sister. =)
Thank you

**Comment by EclipzzeX** [comment]
Speaking for myself, I would not have time to break it down any easier than it already is in the readme.md, but this YouTube Video is pretty specific and is an excellent explanation.  https://www.youtube.com/watch?v=Wl6FVRAj86I&t=301s - Kudos to "Sloth Tech TV"

**Comment by swar** [comment]
Please use Github Discussions next time. This is not an Issue with the software.
